apps/dashboard/src/@/components/blocks/Sidebar.tsx

```diff
@@ -3,24 +3,41 @@ import type React from "react";
 import { cn } from "../../lib/utils";
 import { NavLink } from "../ui/NavLink";
 
-export type SidebarLink = {
+export type SidebarBaseLink = {
   href: string;
   label: React.ReactNode;
   exactMatch?: boolean;
+  hide?: boolean;
```

Comment:
what is this used for, whatever needs to be `hidden` can we just not pass it?

Reply:
Added this because it was very convenient for the Sidebar consumers

Reply:
Removed it and moved filtering logic to consumers